units/Cargo.toml

```diff
@@ -20,6 +20,7 @@ alloc = ["internals/alloc"]
 [dependencies]
 internals = { package = "bitcoin-internals", version = "0.3.0" }
 
+proptest = { version = "=1.2", optional = true }
```

Comment:
Why `=1.2` instead of just plain `1.2`?

Reply:
Yeah, it's definitely wrong to pin dependencies like that.

Reply:
because that's how semver works, although if you're asking me, I worry I am missing something by your question.   if you set this to be 1.2,  the actual resolved version will not always be 1.2, and in this case it would be 1.5 which breaks the MSRV.

Reply:
> Yeah, it's definitely wrong to pin dependencies like that.

Oops I missed this comment somehow.  How should these be pinned?